UnityProject/Assets/Scripts/Items/Implants/Organs/Lungs.cs

```diff
@@ -171,10 +171,15 @@ private bool BreatheIn(GasMix breathGasMix, ReagentMix blood, float efficiency)
 				// Get as much as we need, or as much as in the lungs, whichever is lower
 				Reagent gasReagent = GAS2ReagentSingleton.Instance.GetGasToReagent(Gas.All[i]);
 				float molesRecieved = Mathf.Min(gasMoles, RelatedPart.bloodType.GetSpareGasCapacity(blood, gasReagent));
-				toInhale.Add(gasReagent, molesRecieved * efficiency);
+				if(molesRecieved > 0f)
```

Comment:
```suggestion
				if (Mathf.Approximately(molesRecieved, 0)
```
Perhaps this might be somewhat more useful?